client/blocks/comment-detail/style.scss

```diff
@@ -78,32 +78,70 @@
 	display: flex;
 	flex-flow: row nowrap;
 
+	.comment-detail__header-content {
+		display: flex;
+		flex-wrap: nowrap;
+		width: calc( 100% - 36px );
+	}
+
 	&.is-preview {
 		cursor: pointer;
 		padding: 12px 8px;
 
 		@include breakpoint( "<960px" ) {
-			flex-wrap: wrap;
+			.comment-detail__header-content {
+				flex-wrap: wrap;
+			}
 		}
 	}
 
 	&.is-bulk-edit {
+		.comment-detail__header-content {
+			width: 100%;
+		}
+
 		.comment-detail__checkbox {
-			padding-left: 8px;
+			padding-right: 12px;
 			.form-checkbox {
 				margin: 0;
 			}
 		}
 		.comment-detail__author-preview {
-			padding: 0 12px;
 			.external-link:hover,
 			.external-link:focus {
 				outline: none;
 			}
 		}
 	}
 
-	.button.is-borderless {
+	.comment-detail__action-collapse {
+		.gridicon {
```

Comment:
Not a blocker, but we can pass through a classname to the gridicon so we can reduce CSS specificity here.